packages/back-end/src/controllers/stripe.ts

```diff
@@ -178,11 +255,62 @@ export async function postWebhook(req: Request, res: Response) {
     }
 
     case "customer.subscription.deleted":
+    case "subscription_scheduled.canceled":
     case "customer.subscription.updated": {
+      console.log("updated webhook was hit");
```

Comment:
So, there is one known bug that I need to address. When we're starting a subscription, we're starting it with current number of users + 1, but then, this webhook is getting hit almost immediately after and resetting the billing amount back to current number of users (which is typically 5 in this scenario.) I need to add some check here to prevent that from happening.